Auth0/_ObjectiveManagementAPI.swift

```diff
@@ -69,7 +69,6 @@ public class _ObjectiveManagementAPI: NSObject {
     }
 
     @objc(unlinkUserWithIdentifier:provider:fromUserId:callback:)
-    // swiftlint:disable:next function_parameter_count
```

Comment:
Why are all this lint checks removed?

Reply:
In latest Swiftlint 0.22 they are no longer required.

```
 Line 47: SwiftLint rule 'function_parameter_count' did not trigger a violation in the disabled region. Please remove the disable command.
[03:09:56]: ▸ /Users/distiller/Auth0.swift/Auth0/OAuth2Grant.swift
```